pkg/volume/util/volumepathhandler/volume_path_handler_linux.go

```diff
@@ -148,6 +148,13 @@ func getLoopDeviceFromSysfs(path string) (string, error) {
 	return "", errors.New(ErrDeviceNotFound)
 }
 
+// cleanPath remove any trailing substrings that are not part of the backing file path.
+func cleanBackingFilePath(path string) string {
+	// If the block device was deleted, the path will contain a "(deleted)" suffix
+	path = strings.ReplaceAll(path, "(deleted)", "")
```

Comment:
I'd prefer if `(deleted)` was removed only if it's a suffix. Ie. `TrimSpace(path)`, then `TrimSuffix("(deleted)")` and `TrimSpace` again.